src/Sylius/Component/Core/Customer/Statistics/PerChannelCustomerStatistics.php

```diff
@@ -41,6 +41,10 @@ public function getChannel(): ChannelInterface
 
     public function getAverageOrderValue(): int
     {
+        if (0 == $this->ordersCount) {
```

Comment:
```suggestion
        if (0 === $this->ordersCount) {
```
Sorry, didn't notice this at first. Please use more strict comparison.

Reply:
my bad ;) ty